package.json

```diff
@@ -1,8 +1,8 @@
 {
     "name": "react-native-video",
-    "version": "6.0.0-beta.2",
+    "version": "7.0.0-alpha.19",
     "description": "A <Video /> element for react-native",
-    "main": "lib/index",
```

Comment:
I think this is an error here ?

Reply:
Nope. It's correct way.
We should export "src/index.ts"

Reply:
Oh, my bad.
I didn't know that you export build files.
yep, this should be changed to "lib/index"

Reply:
https://github.com/react-native-video/react-native-video/pull/3204/commits/a813592afa74fab706eea3a0430b405070c0c590

I've finished !